### PR TITLE
refactor: new syntax for `opaque type`

### PR DESCRIPTION
**Pull request by ninaandrup**
Related to #3086

### CODE REVIEW

File: main/test/flix/Test.Dec.Enum.Singleton.flix [hunk 1]
```diff
@@ -1,4 +1,4 @@
-namespace Test/Dec/OpaqueType {
```
Comment (magnus-madsen) [inline main/test/flix/Test.Dec.Enum.Singleton.flix:1]: File should be moved too.

Comment (magnus-madsen) [reply main/test/flix/Test.Dec.Enum.Singleton.flix:1]: Doh. My bad.

Comment (ninaandrup) [reply main/test/flix/Test.Dec.Enum.Singleton.flix:1]: I'm not sure I understand. Should the `Test.Dec.Enum.Singleton.flix` file be moved somewhere?

Comment (magnus-madsen) [reply main/test/flix/Test.Dec.Enum.Singleton.flix:1]: It is already correct. I was just stupid.

**Pull request merged**